config/src/test/java/org/springframework/security/config/annotation/method/configuration/PrePostMethodSecurityConfigurationTests.java

```diff
@@ -453,7 +453,7 @@ public void configureWhenBeanOverridingDisallowedThenWorks() {
 	@WithMockUser(roles = "ADMIN")
 	@Test
 	public void methodSecurityAdminWhenRoleHierarchyBeanAvailableThenUses() {
-		this.spring.register(RoleHierarchyConfig.class, MethodSecurityServiceConfig.class).autowire();
+		this.spring.register(RoleHierarchyConfig.class, MethodSecurityServiceConfig.class, AuthorizationEventPublisherConfig.class).autowire();
```

Comment:
Please place this into a separate test so that this existing test remains the same as before.

Reply:
Done